src/feature/relay/router.c

```diff
@@ -1475,6 +1475,9 @@ router_get_advertised_ipv6_or_ap(const or_options_t *options,
     return;
   }
 
+  log_debug(LD_CONFIG, "Found configured IPv6 ORPort '%s'.",
+            fmt_addrport(addr, port));
+
```

Comment:
So this function can be called quite often and thus we would spam the debug log here a bit too much in my opinion.

I think we might want to log this when we actually parse the ports and after they are sanitized. I think in `port_parse_ports_relay()` could be good after the "check and prune" process.

Reply:
I think I understand what you mean. So, we only want to print the ORPort config that is actually going to be in effect when Tor runs?

If there's no objection I'll make a second PR for this just to keep commit history clean.